Classes/Controller/PostController.php

```diff
@@ -241,6 +243,7 @@ public function newAction(Topic $topic, Post $post = null, Post $quote = null)
         $this->view->assignMultiple([
             'topic' => $topic,
             'post' => $post,
+            'posts' => $posts,
```

Comment:
What's the purpose about `$posts`. IMO `$post` is transferred for being able to quote. Looks like a feature that is not covered by my horizon.

Reply:
under investigation.

Reply:
Even merged already, I still like to get an impression of the purpose here ...